docs/docs/admin/teamforcapella/full-setup/full-setup.md

```diff
@@ -0,0 +1,28 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+This tutorial will guide you through the full setup of a new TeamForCapella
+based project in the Collaboration Manager.
+
+1. Create a new project:
+   [Create a new project](../../../user/projects/create.md)
+1. [Create new tool model](../../../user/projects/models/create.md), select
+   Capella as tool and the version you want to use.
+1. Make sure that right version and nature is selected! If they are not
+   configured, the following steps will not work!
+1. [Link the TeamForCapella repository to the new model](../project-integration/project-integration.md)
+   if not already done.
+1. [Link a Git repository](../../../user/projects/models/create.md#step-31-link-existing-git-repository)
+   if not already done. It's required for the backup pipelines as well as for
+   the diagram cache and model complexity badge.
+1. [Spawn a new persistent session](../../../user/sessions/request.md) with the
+   tool Capella and the version that you chose before.
```

Comment:
I'd probably bold or quote Capella here, maybe slightly clearer as "Spawn a new persistent session. Select **Capella** as the tool, and select the tool version used during step X"

Reply:
I'd avoid to hardcode step numbers because they get outdated quickly. But I updated the sentence a bit.